languages/julia/exercises/concept/leap/.docs/introduction.md

```diff
@@ -4,7 +4,7 @@ Given a year, report if it is a leap year in the Gregorian calendar.
 
 Julia provides a number of [arithmetic operators](https://en.wikipedia.org/wiki/Arithmetic#Arithmetic_operations) for numeric types[^1].
 
-In this exercise you will only need the `%` operator that calculates the [remainder](https://en.wikipedia.org/wiki/Remainder)
+In this exercise you will only need the `%` operator that calculates the [remainder](https://en.wikipedia.org/wiki/Remainder) of the division of two [natural numbers](https://en.wikipedia.org/wiki/Natural_numbers)
```

Comment:
% works with floats (badly), negative numbers, etc.

Reply:
I know but it's not relevant to this exercise.

Would you prefer this?

```suggestion
In this exercise you will only need the `%` operator that calculates the [remainder](https://en.wikipedia.org/wiki/Remainder) of the division of two numbers
```

Reply:
I think so? I just worry we'll confuse people if we say it's for natural numbers.